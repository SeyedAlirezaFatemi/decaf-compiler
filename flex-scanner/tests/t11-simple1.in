{-123-a35,id3a,+*;}[||===!=()&&]<><=>==
    a[24]="7"; n!=if;
false,-if;true32;
forpar
