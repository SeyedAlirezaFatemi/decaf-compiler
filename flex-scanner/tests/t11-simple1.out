{
-
T_INTLITERAL 123
-
T_ID a35
,
T_ID id3a
,
+
*
;
}
[
||
==
=
!=
(
)
&&
]
<
>
<=
>=
=
T_ID a
[
T_INTLITERAL 24
]
=
T_STRINGLITERAL "7"
;
T_ID n
!=
if
;
T_BOOLEANLITERAL false
,
-
if
;
T_ID true32
;
T_ID forpar
